plaidml/exec/ffi.cc

```diff
@@ -122,7 +116,7 @@ plaidml_executable* plaidml_jit(  //
     } else if (jit == "MCJIT") {
       kind = EngineKind::MCJIT;
     }
-    exec->exec = std::make_unique<Executable>(program->program, bufptrs, kind);
+    exec->exec = std::make_unique<Executable>(program->program, std::move(device), bufptrs, kind);
```

Comment:
Why can't we just pass the `device_id` here, what's the point of grabbing a `device` object at this stage?